index.bs

```diff
@@ -3879,13 +3878,26 @@ This [=registration extension=] and [=authentication extension=] enables use of
 :: `uvm`
 
 : Client extension input
-:: The Boolean value true to indicate that this extension is requested by the WebAuthn Relying Party.
+:: The Boolean value `true` to indicate that this extension is requested by the [=[RP]=].
+    <xmp class="idl">
+    partial dictionary AuthenticationExtensionsClientInputs {
+      boolean uvm;
+    };
+    </xmp>
 
 : Client extension processing
 :: None, except creating the authenticator extension input from the client extension input.
 
 : Client extension output
-:: Returns a JSON array of 3-element arrays of numbers that encodes the factors in the authenticator extension output
+:: Returns a JSON array of 3-element arrays of numbers that encodes the factors in the authenticator extension output.
+    <xmp class="idl">
+    typedef sequence<unsigned long> uvmEntry;
+    typedef sequence<uvmEntry> uvmEntries;
```

Comment:
Nit: Should these typedefs' names also have initial uppercase like the others?

Reply:
Done.  Thanks for the careful read @emlun .